src/planner_test.cc

```diff
@@ -209,9 +252,14 @@ static void VerifyCommonExpectations(
   // The joining speeds between segments match.
   for (size_t i = 0; i < segments.size() - 1; ++i) {
     // Let's determine the speed in euclidian space.
+    const size_t m1 = GetDefiningMotor(segments[i]);
+    const size_t m2 = GetDefiningMotor(segments[i + 1]);
+    if (m1 == m2) {
+      EXPECT_EQ(segments[i].v1, segments[i + 1].v0)
+        << "Joining speed between " << i << " and " << (i + 1);
+    }
 #if 1
```

Comment:
I guess we don't need the `#if 1` around here anymore, just leave it always on. It looks like it was `#if`-ed as there were reasons why it failed before.

Reply:
(solved)